src/router.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable arrow-parens */
```

Comment:
A global eslint-disable, that's dangerous. Why do you need this?

Reply:
I never added it, probably it was done automatically somehow

Reply:
Yes, it could be your IDE, mine can also add them when I execute some shortcut.